packages/quantic/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
By curiosity, Is there a perticular reason why we choose apache 2 ?

Reply:
I chose it here because it's the license for the other ui-kit packages. I wasn't part of the original decision but I'd be curious to know. @olamothe could you tell us why this one was chosen?

Reply:
When we first made JSUI public, we asked legal & co, and it was decided that this would be the one we choose for all open source stuff.

Reply:
Thanks!